### ISSUE TITLE
Plugin Settings section disappears when selecting VideoJS v5 player

### ISSUE THREAD

**Issue by jradwan** [Piwigo/piwigo-videojs#90]
After upgrading to 2.8.a, if I select the "VideoJS v5" player, the entire "Plugin Settings" and "VideoJS Plugins" sections disappear so I can't change things like Max Height, Upscale, etc. So when viewing videos through Piwigo, they are shown at full resolution, possibly larger than the browser window can show without scrolling.

**Comment by xbgmsharp** [comment]
That is correct VideoJS v5 is work in progress. As the player is a complete rewrite all plugin must be rewrite as well. In the current master version plugin section plugin are now available. In regards to the resolution work in progress, be continue....

**Comment by jradwan** [comment]
Ok, thanks. I'll still with the v4 player in Piwigo then.

**Comment by xbgmsharp** [comment]
With 2.8.b, the plugin section is available for VideoJS v5. Any feedback appreciate.
The resolution is still work in progress.

**Comment by bigs38** [comment]
Hi,
I have the 2.8.b version of the plug-in and I don't have the section "Plugin préférences" if I select VideoJS v5.

**Comment by xbgmsharp** [comment]
It is correct the "Plugin settings" are not compatible with VideoJS v5, only he "VideoJS plugins" are now available.
The custom CSS and Skin will more likely disappear as it not supported VideoJS v5.

**Comment by bigs38** [comment]
Sorry for my misunderstanding !
But like jradwan, when viewing videos through Piwigo, they are shown larger than the browser window and it's not possible to have acces to play/pause, progress bar..... buttons.
I use Stripped theme, and in this case it's totaly impossible to use VideoJS v5 !
Do you think, that it's possible to "auto-adjust" the size of the vidéo with a div or something else ?

**Comment by xbgmsharp** [comment]
Correct the "upscale" and "Max Height " feature are pending implementation.

**Comment by xbgmsharp** [comment]
Not all videojs plugin are supported across all version. Hence some menu that disappear.
Please feel free to to test and report isf those videojs plugin works fine.

**Comment by EddyLB** [comment]
Fixed by now and the Native Browser is the recommended choice today.
